From: Jan-Ivar Bruaroey <wingboy@gmail.com>
Date: Thu, 29 Jun 2023 21:13:44 -0400
Subject: (tmp-cherry-pick) Revert "stats: remove RTCRtpInboundRTPStream and
 RTCRtpoutboundRTPStream aliases" (4463ff0296)
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

This reverts commit 9671d60925b81baefd4a0d6b05ad539fa4a782d7.

Reason for revert: Breaks dependencies, will re-land after fixes

Original change's description:
> stats: remove RTCRtpInboundRTPStream and RTCRtpoutboundRTPStream aliases
>
> after upgrading downstream projects
>
> BUG=webrtc:14973
>
> Change-Id: I5df8e95a1c70b1d6078e255166c36ed01f868b6a
> Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/296820
> Reviewed-by: Christoffer Jansson <jansson@webrtc.org>
> Reviewed-by: Henrik Boström <hbos@webrtc.org>
> Commit-Queue: Philipp Hancke <phancke@microsoft.com>
> Cr-Commit-Position: refs/heads/main@{#39526}

Bug: webrtc:14973
Change-Id: I50878526566660d9772f7c8664970eec8bd86341
No-Presubmit: true
No-Tree-Checks: true
No-Try: true
Reviewed-on: https://webrtc-review.googlesource.com/c/src/+/296940
Reviewed-by: Philipp Hancke <phancke@microsoft.com>
Commit-Queue: Philipp Hancke <phancke@microsoft.com>
Bot-Commit: rubber-stamper@appspot.gserviceaccount.com <rubber-stamper@appspot.gserviceaccount.com>
Auto-Submit: Henrik Boström <hbos@webrtc.org>
Cr-Commit-Position: refs/heads/main@{#39530}
---
 api/stats/rtcstats_objects.h                      | 4 ++++
 pc/peer_connection_mediachannel_split_unittest.cc | 4 ++--
 2 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/api/stats/rtcstats_objects.h b/api/stats/rtcstats_objects.h
index 790b3b1351..b4b272d2af 100644
--- a/api/stats/rtcstats_objects.h
+++ b/api/stats/rtcstats_objects.h
@@ -493,6 +493,8 @@ class RTC_EXPORT RTCInboundRtpStreamStats final
   // The former googMinPlayoutDelayMs (in seconds).
   RTCNonStandardStatsMember<double> min_playout_delay;
 };
+// TODO(bugs.webrtc.org/14973): remove name alias.
+using RTCInboundRTPStreamStats = RTCInboundRtpStreamStats;
 
 // https://w3c.github.io/webrtc-stats/#outboundrtpstats-dict*
 class RTC_EXPORT RTCOutboundRtpStreamStats final
@@ -544,6 +546,8 @@ class RTC_EXPORT RTCOutboundRtpStreamStats final
       power_efficient_encoder;
   RTCStatsMember<std::string> scalability_mode;
 };
+// TODO(bugs.webrtc.org/14973): remove name alias.
+using RTCOutboundRTPStreamStats = RTCOutboundRtpStreamStats;
 
 // https://w3c.github.io/webrtc-stats/#remoteinboundrtpstats-dict*
 class RTC_EXPORT RTCRemoteInboundRtpStreamStats final
diff --git a/pc/peer_connection_mediachannel_split_unittest.cc b/pc/peer_connection_mediachannel_split_unittest.cc
index a6b9640269..3eca104add 100644
--- a/pc/peer_connection_mediachannel_split_unittest.cc
+++ b/pc/peer_connection_mediachannel_split_unittest.cc
@@ -48,7 +48,7 @@ class PeerConnectionMediaChannelSplitTest
 
 int NacksReceivedCount(PeerConnectionIntegrationWrapper& pc) {
   rtc::scoped_refptr<const webrtc::RTCStatsReport> report = pc.NewGetStats();
-  auto sender_stats = report->GetStatsOfType<RTCOutboundRtpStreamStats>();
+  auto sender_stats = report->GetStatsOfType<RTCOutboundRTPStreamStats>();
   if (sender_stats.size() != 1) {
     ADD_FAILURE();
     return 0;
@@ -61,7 +61,7 @@ int NacksReceivedCount(PeerConnectionIntegrationWrapper& pc) {
 
 int NacksSentCount(PeerConnectionIntegrationWrapper& pc) {
   rtc::scoped_refptr<const webrtc::RTCStatsReport> report = pc.NewGetStats();
-  auto receiver_stats = report->GetStatsOfType<RTCInboundRtpStreamStats>();
+  auto receiver_stats = report->GetStatsOfType<RTCInboundRTPStreamStats>();
   if (receiver_stats.size() != 1) {
     ADD_FAILURE();
     return 0;
-- 
2.37.1 (Apple Git-137.1)

